llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -851,7 +852,29 @@ TableFiles processOneModule(std::unique_ptr<Module> M, bool IsEsimd,
   bool IsLLVMUsedRemoved = false;
   if (GlobalVariable *GV = M->getGlobalVariable("llvm.used")) {
```

Comment:
Could this removal procedure be moved into a separate function? Just to simplify reading of processOneModule function.

Reply:
Makes sense, thank you, I'll move it to a separate function.

Reply:
Fixed in ffeac5eb614a56894c03433c4bf885fa60219dc3